src/arvgcconverter.c

```diff
@@ -542,3 +550,29 @@ arv_gc_converter_get_unit (ArvGcConverter *gc_converter, GError **error)
 
 	return arv_gc_property_node_get_string (ARV_GC_PROPERTY_NODE (priv->unit), error);
 }
+
+ArvGcDisplayNotation
+arv_gc_converter_get_display_notation (ArvGcConverter *gc_converter, GError **error)
+{
+	ArvGcConverterPrivate *priv = arv_gc_converter_get_instance_private (gc_converter);
+
+	g_return_val_if_fail (ARV_IS_GC_CONVERTER (gc_converter), ARV_GC_DISPLAY_NOTATION_AUTOMATIC);
+
+	if (priv->display_notation == NULL)
+		return ARV_GC_DISPLAY_NOTATION_AUTOMATIC;
+
+	return arv_gc_property_node_get_display_notation (ARV_GC_PROPERTY_NODE (priv->display_notation), ARV_GC_DISPLAY_NOTATION_AUTOMATIC);
+}
+
+gint64
+arv_gc_converter_get_display_precision (ArvGcConverter *gc_converter, GError **error)
+{
+	ArvGcConverterPrivate *priv = arv_gc_converter_get_instance_private (gc_converter);
+
+	g_return_val_if_fail (ARV_IS_GC_CONVERTER (gc_converter), 6);
```

Comment:
Please use a define for default display precision value.

Reply:
Thank you for reviewing the commit. I agree that having defines for default values is really necessary.

I added a new commit which creates a new `arvgcdefaults.h` file for GenICam default values. Let me know if you would like to see the default values implemented in some other way.

Reply:
A new file with defaults is a good idea, but I'd prefer to not make these defines public, at least for now. Please rename it as arvgcdefaultsprivate.h.

Reply:
Yes, good point. I fixed the name and meson/header related code.